### DISCUSSION TITLE
Clearing `.svg` and `.png` from unnecessary content

### DISCUSSION THREAD

**Discussion by CanadianBeaver** [facebook/docusaurus#10291] [Ideas]
Many software puts in `.svg`, `.png` and other images the content that is not needed. F.e. Fooocus and Draw.io keeps inside images their editor data.

Idea: let's `docusaurus` empties images during build and publishes lighted images to web-site.

Thank you

**Comment by slorber** [comment]
I didn't have time to take a look yet, but this PR from SVGO author @SethFalco could probably solve this problem?

https://github.com/facebook/docusaurus/pull/10211
